packages/xod-pm/src/messages.js

```diff
@@ -1,4 +1,5 @@
 export const CANT_PARSE_LIBRARY_REQUEST = 'Canʼt parse request to fetch library data';
+export const NO_NETWORK_CONNECTION = 'No network connection';
```

Comment:
Maybe the traditional “Service is unavailable” is a more precise wording?

Perhaps the network connection is up, but our site is down.

Reply:
In that case, we'll get not an Error, we'll get an object with fields like `status`, `body` and etc. So we can detect network error and HTTP errors...

But it's really strange that this message appears only here, not everywhere where we can get the network error. So I'll replace it, but, I hope, in the future we could separate error "Service is unavailable" and "No Internet connection" :)